{
  "version": "4.38.0",
  "time": 407.94,
  "errors": [],
  "tasks": [
    {
      "resource": {
        "name": "ft_compras",
        "title": "Compras",
        "path": "data/ft_compras.csv.gz",
        "schema": {
          "fields": [
            {
              "name": "id_tempo",
              "title": "Tempo - Chave",
              "type": "integer",
              "format": "default",
              "description": "- > \n  Chave estrangeira (Foreign Key) que aponta para a Chave Primária (Primary keys) da tabela dm_tempo_diario. \n  Ou seja, por meio dessa chave é possível identificar a data completa a qual o dado se refere.\n\n  Nota: \n  No site dados.mg.gov.br a tabela dm_tempo_diario pode ser encontrada no recurso Calendários - Dias \n  de cada conjunto.\n",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_procedimento",
              "title": "Procedimento de Contratação - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_orgao_demanda",
              "title": "Órgão Demandante - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_orgao_contrato",
              "title": "Órgão do Contrato - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_situacao_proc",
              "title": "Situação do Processo de Compra - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_situacao_cont",
              "title": "Situação do Contrato - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_municipio",
              "title": "Município - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_contratado",
              "title": "Contratado - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_tipo_licitacao",
              "title": "Tipo de Licitação - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_grupo_matserv",
              "title": "Grupo Material e Serviço - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_classe_matserv",
              "title": "Classe Material e Serviço - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_material_servico",
              "title": "Material e Serviço - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_item_matserv",
              "title": "Item Material e Serviço - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_processo",
              "title": "Processo - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_contrato",
              "title": "Número do Contrato - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_unidade_medida",
              "title": "Unidade de Medida - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_linha_fornec",
              "title": "Linha de Fornecimento - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_item",
              "title": "Item - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_unidade_orc",
              "title": "Unidade Orçamentária - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "unique": false
              }
            },
            {
              "name": "ano_particao",
              "title": "Ano Partição",
              "type": "year",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true
              }
            },
            {
              "name": "dt_item_homologa",
              "title": "Data da Homologação do Item",
              "type": "date",
              "format": "default",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "qt_item_pedido",
              "title": "Item do Pedido - Quantidade",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ".",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_un_referencia",
              "title": "Valor Unitário de Referência",
              "description": "Valor unitário estimado de um determinado bem ou serviço, obtido por meio de pesquisa de mercado ou cadastro de preços.\n",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ".",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_referencia",
              "title": "Valor de Referência",
              "description": "Valor estimado de um determinado bem ou serviço, obtido por meio de pesquisa de mercado ou cadastro de preços.\n",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ".",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_un_homologado",
              "title": "Valor Unitário Homologado",
              "description": "Valor unitário final licitado ou comprado.",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ".",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_homologado",
              "title": "Valor Homologado",
              "description": "Valor final licitado ou comprado.",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ".",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_atualizado",
              "title": "Valor Atualizado",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ".",
              "constraints": {
                "required": true
              }
            }
          ],
          "missingValues": [
            ""
          ],
          "foreignKeys": [
            {
              "fields": [
                "id_processo"
              ],
              "reference": {
                "fields": [
                  "id_processo"
                ],
                "resource": "dm_processo"
              }
            },
            {
              "fields": [
                "id_situacao_cont"
              ],
              "reference": {
                "fields": [
                  "id_situacao_cont"
                ],
                "resource": "dm_situacao_cont"
              }
            },
            {
              "fields": [
                "id_situacao_proc"
              ],
              "reference": {
                "fields": [
                  "id_situacao_proc"
                ],
                "resource": "dm_situacao_proc"
              }
            },
            {
              "fields": [
                "id_linha_fornec"
              ],
              "reference": {
                "fields": [
                  "id_linha_fornec"
                ],
                "resource": "dm_linha_fornec"
              }
            },
            {
              "fields": [
                "id_classe_matserv"
              ],
              "reference": {
                "fields": [
                  "id_classe_matserv"
                ],
                "resource": "dm_classe_matserv"
              }
            },
            {
              "fields": [
                "id_tempo"
              ],
              "reference": {
                "fields": [
                  "id_tempo"
                ],
                "resource": "dm_tempo_diario"
              }
            },
            {
              "fields": [
                "id_municipio"
              ],
              "reference": {
                "fields": [
                  "id_municipio"
                ],
                "resource": "dm_municipio"
              }
            },
            {
              "fields": [
                "id_item_matserv"
              ],
              "reference": {
                "fields": [
                  "id_item_matserv"
                ],
                "resource": "dm_item_matserv"
              }
            },
            {
              "fields": [
                "id_tipo_licitacao"
              ],
              "reference": {
                "fields": [
                  "id_tipo_licitacao"
                ],
                "resource": "dm_tipo_licitacao"
              }
            },
            {
              "fields": [
                "id_item"
              ],
              "reference": {
                "fields": [
                  "id_item"
                ],
                "resource": "dm_item_desp"
              }
            },
            {
              "fields": [
                "id_material_servico"
              ],
              "reference": {
                "fields": [
                  "id_material_servico"
                ],
                "resource": "dm_material_servico"
              }
            },
            {
              "fields": [
                "id_orgao_contrato"
              ],
              "reference": {
                "fields": [
                  "id_orgao_contrato"
                ],
                "resource": "dm_orgao_contrato"
              }
            },
            {
              "fields": [
                "id_contrato"
              ],
              "reference": {
                "fields": [
                  "id_contrato"
                ],
                "resource": "dm_contrato"
              }
            },
            {
              "fields": [
                "id_orgao_demanda"
              ],
              "reference": {
                "fields": [
                  "id_orgao_demanda"
                ],
                "resource": "dm_orgao_demanda"
              }
            },
            {
              "fields": [
                "id_unidade_medida"
              ],
              "reference": {
                "fields": [
                  "id_unidade_medida"
                ],
                "resource": "dm_unidade_medida"
              }
            },
            {
              "fields": [
                "id_grupo_matserv"
              ],
              "reference": {
                "fields": [
                  "id_grupo_matserv"
                ],
                "resource": "dm_grupo_matserv"
              }
            },
            {
              "fields": [
                "id_contratado"
              ],
              "reference": {
                "fields": [
                  "id_contratado"
                ],
                "resource": "dm_contratado"
              }
            },
            {
              "fields": [
                "id_procedimento"
              ],
              "reference": {
                "fields": [
                  "id_procedimento"
                ],
                "resource": "dm_procedimento"
              }
            }
          ]
        },
        "sources": [
          {
            "title": "Tabela origem no schema AGE7 no banco de dados do Portal da Transparência",
            "table": "ft_compras"
          }
        ],
        "format": "csv",
        "mediatype": "text/csv",
        "encoding": "utf-8-sig",
        "dialect": {
          "delimiter": ";"
        },
        "profile": "tabular-data-resource",
        "scheme": "file",
        "hashing": "md5",
        "compression": "gz",
        "stats": {
          "hash": "e2b7ce1328f32a2668e7935c17252012",
          "bytes": 58473520,
          "fields": 27,
          "rows": 1919464
        }
      },
      "time": 407.94,
      "scope": [
        "hash-count-error",
        "byte-count-error",
        "field-count-error",
        "row-count-error",
        "blank-header",
        "extra-label",
        "missing-label",
        "blank-label",
        "duplicate-label",
        "incorrect-label",
        "blank-row",
        "primary-key-error",
        "foreign-key-error",
        "extra-cell",
        "missing-cell",
        "type-error",
        "constraint-error",
        "unique-error"
      ],
      "partial": false,
      "errors": [
        {
          "cells": [
            "43520",
            "2",
            "1831030",
            "27",
            "1",
            "1",
            "66",
            "22000",
            "1",
            "77",
            "73",
            "9726",
            "92493",
            "384217",
            "79252",
            "889",
            "1831901",
            "2623",
            "2270",
            "2022",
            "2022-12-14",
            "0.0000",
            "207894.5600",
            "207894.56",
            "207894.5600",
            "207894.56",
            "207894.56"
          ],
          "rowNumber": 1821800,
          "rowPosition": 1821801,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79252\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1821801\" violates the foreign key: for \"id_contrato\": values \"79252\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43520",
            "2",
            "1831030",
            "27",
            "1",
            "1",
            "338",
            "22000",
            "1",
            "77",
            "73",
            "9726",
            "92493",
            "384193",
            "79247",
            "889",
            "1831901",
            "2623",
            "2270",
            "2022",
            "2022-12-14",
            "0.0000",
            "83157.8200",
            "83157.82",
            "83157.8200",
            "83157.82",
            "83157.82"
          ],
          "rowNumber": 1821808,
          "rowPosition": 1821809,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79247\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1821809\" violates the foreign key: for \"id_contrato\": values \"79247\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43520",
            "2",
            "1831030",
            "27",
            "1",
            "1",
            "66",
            "22000",
            "1",
            "77",
            "73",
            "9726",
            "159483",
            "384404",
            "79251",
            "889",
            "1831901",
            "2623",
            "2270",
            "2022",
            "2022-12-16",
            "0.0000",
            "19258.8000",
            "19258.80",
            "19258.8000",
            "19258.80",
            "19258.80"
          ],
          "rowNumber": 1824128,
          "rowPosition": 1824129,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79251\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1824129\" violates the foreign key: for \"id_contrato\": values \"79251\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43520",
            "2",
            "1831030",
            "27",
            "1",
            "1",
            "338",
            "22000",
            "1",
            "77",
            "73",
            "9726",
            "159483",
            "384423",
            "79253",
            "889",
            "1831901",
            "2623",
            "2270",
            "2022",
            "2022-12-16",
            "0.0000",
            "7703.5200",
            "7703.52",
            "7703.5200",
            "7703.52",
            "7703.52"
          ],
          "rowNumber": 1824147,
          "rowPosition": 1824148,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79253\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1824148\" violates the foreign key: for \"id_contrato\": values \"79253\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44419",
            "4",
            "1831008",
            "5",
            "1",
            "1",
            "883",
            "21034",
            "1",
            "77",
            "183",
            "4381",
            "93974",
            "386160",
            "79254",
            "889",
            "1831800",
            "3136",
            "2925",
            "2022",
            "2023-01-19",
            "0.0000",
            "100000.0000",
            "100000.00",
            "69900.0000",
            "69900.00",
            "69900.00"
          ],
          "rowNumber": 1916516,
          "rowPosition": 1916517,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79254\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1916517\" violates the foreign key: for \"id_contrato\": values \"79254\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44495",
            "15",
            "1831019",
            "10",
            "1",
            "1",
            "883",
            "19695",
            "1",
            "39",
            "712",
            "7131",
            "161473",
            "386249",
            "79248",
            "889",
            "1831925",
            "3067",
            "2000",
            "2023",
            "2023-01-25",
            "6.0000",
            "47.0000",
            "282.00",
            "47.0000",
            "282.00",
            "282.00"
          ],
          "rowNumber": 1916661,
          "rowPosition": 1916662,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1916662\" violates the foreign key: for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44495",
            "15",
            "1831019",
            "10",
            "1",
            "1",
            "883",
            "19695",
            "1",
            "39",
            "712",
            "20379",
            "161454",
            "386249",
            "79248",
            "889",
            "1831925",
            "3067",
            "2000",
            "2023",
            "2023-01-25",
            "2.0000",
            "48.0000",
            "96.00",
            "48.0000",
            "96.00",
            "96.00"
          ],
          "rowNumber": 1916662,
          "rowPosition": 1916663,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1916663\" violates the foreign key: for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44495",
            "15",
            "1831019",
            "10",
            "1",
            "1",
            "883",
            "19695",
            "1",
            "39",
            "712",
            "7131",
            "265542",
            "386249",
            "79248",
            "889",
            "1831925",
            "3067",
            "2000",
            "2023",
            "2023-01-25",
            "1.0000",
            "20.0000",
            "20.00",
            "20.0000",
            "20.00",
            "20.00"
          ],
          "rowNumber": 1916663,
          "rowPosition": 1916664,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1916664\" violates the foreign key: for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44495",
            "15",
            "1831019",
            "10",
            "1",
            "1",
            "883",
            "19695",
            "1",
            "39",
            "712",
            "20379",
            "265551",
            "386249",
            "79248",
            "889",
            "1831925",
            "3067",
            "2000",
            "2023",
            "2023-01-25",
            "1.0000",
            "20.0000",
            "20.00",
            "20.0000",
            "20.00",
            "20.00"
          ],
          "rowNumber": 1916664,
          "rowPosition": 1916665,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1916665\" violates the foreign key: for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44495",
            "15",
            "1831019",
            "10",
            "1",
            "1",
            "883",
            "19695",
            "1",
            "39",
            "712",
            "20379",
            "100999",
            "386249",
            "79248",
            "889",
            "1831925",
            "3067",
            "2000",
            "2023",
            "2023-01-25",
            "2.0000",
            "28.0000",
            "56.00",
            "28.0000",
            "56.00",
            "56.00"
          ],
          "rowNumber": 1916665,
          "rowPosition": 1916666,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1916666\" violates the foreign key: for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44495",
            "15",
            "1831019",
            "10",
            "1",
            "1",
            "883",
            "19695",
            "1",
            "39",
            "712",
            "20379",
            "346252",
            "386249",
            "79248",
            "889",
            "1831925",
            "3067",
            "2000",
            "2023",
            "2023-01-25",
            "1.0000",
            "28.0000",
            "28.00",
            "28.0000",
            "28.00",
            "28.00"
          ],
          "rowNumber": 1916666,
          "rowPosition": 1916667,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1916667\" violates the foreign key: for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44495",
            "15",
            "1831019",
            "10",
            "1",
            "1",
            "883",
            "19695",
            "1",
            "39",
            "712",
            "7131",
            "100994",
            "386249",
            "79248",
            "889",
            "1831925",
            "3067",
            "2000",
            "2023",
            "2023-01-25",
            "2.0000",
            "20.0000",
            "40.00",
            "20.0000",
            "40.00",
            "40.00"
          ],
          "rowNumber": 1916667,
          "rowPosition": 1916668,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1916668\" violates the foreign key: for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44495",
            "15",
            "1831019",
            "10",
            "1",
            "1",
            "883",
            "19695",
            "1",
            "39",
            "712",
            "20379",
            "161459",
            "386249",
            "79248",
            "889",
            "1831925",
            "3067",
            "2000",
            "2023",
            "2023-01-25",
            "4.0000",
            "25.0000",
            "100.00",
            "25.0000",
            "100.00",
            "100.00"
          ],
          "rowNumber": 1916668,
          "rowPosition": 1916669,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1916669\" violates the foreign key: for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44495",
            "15",
            "1831019",
            "10",
            "1",
            "1",
            "883",
            "19695",
            "1",
            "39",
            "712",
            "7131",
            "171415",
            "386249",
            "79248",
            "889",
            "1831925",
            "3067",
            "2000",
            "2023",
            "2023-01-25",
            "4.0000",
            "20.0000",
            "80.00",
            "20.0000",
            "80.00",
            "80.00"
          ],
          "rowNumber": 1916669,
          "rowPosition": 1916670,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1916670\" violates the foreign key: for \"id_contrato\": values \"79248\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44497",
            "14",
            "1831034",
            "50",
            "1",
            "8",
            "637",
            "73597",
            "1",
            "86",
            "736",
            "21535",
            "169158",
            "386304",
            "46296",
            "889",
            "1831184",
            "2564",
            "1760",
            "2023",
            "2023-01-27",
            "3.0000",
            "3300.0000",
            "9900.00",
            "3066.0000",
            "9198.00",
            "9198.00"
          ],
          "rowNumber": 1917266,
          "rowPosition": 1917267,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contratado\": values \"73597\" not found in the lookup table \"dm_contratado\" as \"id_contratado\"",
          "message": "Row at position \"1917267\" violates the foreign key: for \"id_contratado\": values \"73597\" not found in the lookup table \"dm_contratado\" as \"id_contratado\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44490",
            "15",
            "1831058",
            "54",
            "1",
            "1",
            "883",
            "19695",
            "1",
            "39",
            "712",
            "20379",
            "161459",
            "386200",
            "79249",
            "889",
            "1831925",
            "3067",
            "2012",
            "2023",
            "2023-01-23",
            "2.0000",
            "25.0000",
            "50.00",
            "25.0000",
            "50.00",
            "50.00"
          ],
          "rowNumber": 1918323,
          "rowPosition": 1918324,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79249\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1918324\" violates the foreign key: for \"id_contrato\": values \"79249\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44490",
            "15",
            "1831058",
            "54",
            "1",
            "1",
            "883",
            "19695",
            "1",
            "39",
            "712",
            "20379",
            "100999",
            "386200",
            "79249",
            "889",
            "1831925",
            "3067",
            "2012",
            "2023",
            "2023-01-23",
            "4.0000",
            "28.0000",
            "112.00",
            "28.0000",
            "112.00",
            "112.00"
          ],
          "rowNumber": 1918324,
          "rowPosition": 1918325,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79249\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1918325\" violates the foreign key: for \"id_contrato\": values \"79249\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44490",
            "15",
            "1831058",
            "54",
            "1",
            "1",
            "883",
            "19695",
            "1",
            "39",
            "712",
            "7131",
            "161473",
            "386200",
            "79249",
            "889",
            "1831925",
            "3067",
            "2012",
            "2023",
            "2023-01-23",
            "10.0000",
            "47.0000",
            "470.00",
            "47.0000",
            "470.00",
            "470.00"
          ],
          "rowNumber": 1918325,
          "rowPosition": 1918326,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79249\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1918326\" violates the foreign key: for \"id_contrato\": values \"79249\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44490",
            "15",
            "1831058",
            "54",
            "1",
            "1",
            "883",
            "19695",
            "1",
            "39",
            "712",
            "20379",
            "161454",
            "386200",
            "79249",
            "889",
            "1831925",
            "3067",
            "2012",
            "2023",
            "2023-01-23",
            "1.0000",
            "48.0000",
            "48.00",
            "48.0000",
            "48.00",
            "48.00"
          ],
          "rowNumber": 1918326,
          "rowPosition": 1918327,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79249\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1918327\" violates the foreign key: for \"id_contrato\": values \"79249\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44490",
            "15",
            "1831058",
            "54",
            "1",
            "1",
            "883",
            "19695",
            "1",
            "39",
            "712",
            "7131",
            "171415",
            "386200",
            "79249",
            "889",
            "1831925",
            "3067",
            "2012",
            "2023",
            "2023-01-23",
            "100.0000",
            "20.0000",
            "2000.00",
            "20.0000",
            "2000.00",
            "2000.00"
          ],
          "rowNumber": 1918327,
          "rowPosition": 1918328,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79249\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1918328\" violates the foreign key: for \"id_contrato\": values \"79249\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43487",
            "4",
            "1831016",
            "16",
            "1",
            "1",
            "66",
            "37796",
            "1",
            "39",
            "97",
            "12842",
            "134530",
            "386016",
            "79250",
            "26",
            "1831812",
            "3065",
            "2082",
            "2022",
            "2023-01-05",
            "0.0000",
            "166940.0000",
            "166940.00",
            "119000.0000",
            "119000.00",
            "119000.00"
          ],
          "rowNumber": 1919103,
          "rowPosition": 1919104,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79250\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1919104\" violates the foreign key: for \"id_contrato\": values \"79250\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43487",
            "4",
            "1831016",
            "16",
            "1",
            "1",
            "66",
            "37796",
            "1",
            "39",
            "97",
            "12842",
            "134531",
            "386016",
            "79250",
            "26",
            "1831812",
            "3065",
            "2082",
            "2022",
            "2023-01-05",
            "0.0000",
            "186580.0000",
            "186580.00",
            "119000.0000",
            "119000.00",
            "119000.00"
          ],
          "rowNumber": 1919104,
          "rowPosition": 1919105,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79250\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1919105\" violates the foreign key: for \"id_contrato\": values \"79250\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43487",
            "4",
            "1831016",
            "16",
            "1",
            "1",
            "66",
            "37796",
            "1",
            "39",
            "97",
            "12842",
            "134529",
            "386016",
            "79250",
            "26",
            "1831812",
            "3065",
            "2082",
            "2022",
            "2023-01-05",
            "0.0000",
            "166940.0000",
            "166940.00",
            "178500.0000",
            "178500.00",
            "178500.00"
          ],
          "rowNumber": 1919105,
          "rowPosition": 1919106,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79250\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1919106\" violates the foreign key: for \"id_contrato\": values \"79250\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43487",
            "4",
            "1831016",
            "16",
            "1",
            "1",
            "66",
            "37796",
            "1",
            "39",
            "97",
            "12842",
            "134532",
            "386016",
            "79250",
            "26",
            "1831812",
            "3065",
            "2082",
            "2022",
            "2023-01-05",
            "0.0000",
            "154174.0000",
            "154174.00",
            "154700.0000",
            "154700.00",
            "154700.00"
          ],
          "rowNumber": 1919106,
          "rowPosition": 1919107,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79250\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1919107\" violates the foreign key: for \"id_contrato\": values \"79250\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43487",
            "4",
            "1831016",
            "16",
            "1",
            "1",
            "66",
            "37796",
            "1",
            "39",
            "97",
            "12842",
            "149996",
            "386016",
            "79250",
            "26",
            "1831812",
            "3065",
            "2082",
            "2022",
            "2023-01-05",
            "0.0000",
            "41280.0000",
            "41280.00",
            "47600.0000",
            "47600.00",
            "47600.00"
          ],
          "rowNumber": 1919107,
          "rowPosition": 1919108,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79250\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1919108\" violates the foreign key: for \"id_contrato\": values \"79250\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43487",
            "4",
            "1831016",
            "16",
            "1",
            "1",
            "66",
            "37796",
            "1",
            "39",
            "97",
            "12842",
            "149998",
            "386016",
            "79250",
            "26",
            "1831812",
            "3065",
            "2082",
            "2022",
            "2023-01-05",
            "0.0000",
            "196400.0000",
            "196400.00",
            "214200.0000",
            "214200.00",
            "214200.00"
          ],
          "rowNumber": 1919108,
          "rowPosition": 1919109,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79250\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1919109\" violates the foreign key: for \"id_contrato\": values \"79250\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43487",
            "4",
            "1831016",
            "16",
            "1",
            "1",
            "66",
            "37796",
            "1",
            "39",
            "97",
            "12842",
            "134527",
            "386016",
            "79250",
            "26",
            "1831812",
            "3065",
            "2082",
            "2022",
            "2023-01-05",
            "0.0000",
            "217940.0000",
            "217940.00",
            "119000.0000",
            "119000.00",
            "119000.00"
          ],
          "rowNumber": 1919109,
          "rowPosition": 1919110,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79250\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1919110\" violates the foreign key: for \"id_contrato\": values \"79250\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43487",
            "4",
            "1831016",
            "16",
            "1",
            "1",
            "66",
            "37796",
            "1",
            "39",
            "97",
            "12842",
            "134528",
            "386016",
            "79250",
            "26",
            "1831812",
            "3065",
            "2082",
            "2022",
            "2023-01-05",
            "0.0000",
            "153192.0000",
            "153192.00",
            "119000.0000",
            "119000.00",
            "119000.00"
          ],
          "rowNumber": 1919110,
          "rowPosition": 1919111,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79250\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1919111\" violates the foreign key: for \"id_contrato\": values \"79250\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43508",
            "4",
            "1831081",
            "77",
            "1",
            "1",
            "66",
            "16130",
            "1",
            "42",
            "399",
            "8725",
            "168584",
            "386211",
            "79246",
            "889",
            "1831334",
            "2923",
            "2014",
            "2022",
            "2023-01-24",
            "168.0000",
            "1625.0000",
            "273000.00",
            "1625.0000",
            "273000.00",
            "273000.00"
          ],
          "rowNumber": 1919374,
          "rowPosition": 1919375,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"79246\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1919375\" violates the foreign key: for \"id_contrato\": values \"79246\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        }
      ],
      "stats": {
        "errors": 29
      },
      "valid": false
    }
  ],
  "stats": {
    "errors": 29,
    "tasks": 1
  },
  "valid": false
}